### ISSUE TITLE
Any chance of a figwheel example ?

### ISSUE THREAD

**Issue by olymk2** [djblue/portal#36]
I have added the requirement to deps.edn, but i am not sure how to make the web app launch, the example uses -m "cljs.main" but in figwheel you don't normally do this.

In my dev.cljs.edn file I have this config which has a load of extra mains, I figure I need to add an entry point in there tried a few things, is this the right way or should I be launching it from the repl in the user.cljs namespace perhaps ?

```
^{:extra-main-files
  {:devcards {:main cards.core}
   :portal {:main portal.web}
   :testing {:main website.test-server}}
  :auto-testing true
  }
{:output-to "resources/public/cljs-out/dev-main.js"
 :closure-defines {website.state/environment "development"}
 :optimizations :none
 :pretty-print true
 :source-map true
 :source-map-timestamp true
 :devcards true
 :main website.core}
```

**Comment by olymk2** [comment]
I am not sure I thought extra mains was a bit like that. 
looking at that example I guess I can add user.main as an extra main and use what you have in cljs.user thats what I was not sure about, I thought portal would just pop up but seems you have to add the code to launch it, I thought that was only for extra customization I will try that way and let you know if that works.

**Comment by olymk2** [comment]
I have tried in both firefox and chrome but both give me a white screen, whats this code doing ? 
```
(defn swap-dev []
  (set! js/portal.runtime.web.launcher.code_url
        (str js/window.location.origin "/main.js")))
```
Is main.js my reagent app js ie resources/public/cljs-out/dev-main.js ?

or part of portal, either way I get 

Loading failed for the <script> with source “http://localhost:9500/main.js”

**Comment by djblue** [comment]
That is for swapping to the dev version of portal.web. You don't want that. I would delete most of the user.cljs as most of it is specific to development in portal.

**Comment by olymk2** [comment]
@djblue thanks for the help, so basically add the dependency then just launch (p/open) from user.cljs or where ever to trigger the interface.

I will play a bit more but that seems to have done it, thanks again for your time :)

**Comment by djblue** [comment]
I'm glad you could get it working! Do mind adding a minimal fighweel project under examples or providing me some details so I can do so?

**Comment by olymk2** [comment]
Give me a day or two and i will see about adding one or giving some details its late here and i want to play a bit first :)

**Comment by olymk2** [comment]
https://github.com/djblue/portal/pull/37 
not figured out if I can make portal run when you jack in, but this at least works if you eval the commands.

**Comment by djblue** [comment]
Resolved with #37, thanks again for the PR!
